### ISSUE TITLE
Training details

### ISSUE THREAD

**Issue by whyandbecause** [mahmoodlab/CLAM#137]
Hi，thank you for your excellent work, I was confused with training details. In your paper, you said " During training, patches are randomly sampled from slides in the training set using a batch size of 512.", but from the code, you trained model by slide_level h5 files pre-extracted by run extract_features_fp.py , and batch_size=1.  

Thank you for your reply!

**Comment by fedshyvana** [comment]
i think you're looking at the training details section for assuming same label for each patch as the slide. For both MIL and CLAM we use batchsize of 1 which is clearly stated in the paper.

**Comment by fedshyvana** [comment]
i think you're looking at the training details section for assuming same label for each patch as the slide. For both MIL and CLAM we use batchsize of 1 which is clearly stated in the paper.
